src/app/App.tsx

```diff
@@ -1,50 +1,61 @@
-import React, { useState } from 'react';
+import React, { useState, useEffect } from 'react';
 import { BrowserRouter as Router } from 'react-router-dom';
 import { I18nextProvider } from 'react-i18next';
-import { ApolloClient, InMemoryCache, ApolloProvider } from '@apollo/client';
 import '@patternfly/react-core/dist/styles/base.css';
 import { AppLayout, Loading } from '@app/components';
 import { AppRoutes } from '@app/routes';
 import { initI18N } from '@i18n/i18n';
 import { KeycloakAuthProvider, KeycloakContext, getKeycloakInstance } from '@app/auth/keycloak';
 import '@app/app.css';
+import { useVerifyTokenQuery } from '@app/models';
 
 let keycloak: Keycloak.KeycloakInstance | undefined;
+let tokenresponse: string | boolean | unknown;
 
 const App: React.FunctionComponent = () => {
   const [initialized, setInitialized] = useState(false);
-  const client = new ApolloClient({
-    uri: process.env.GRAPHQL_ENDPOINT || 'http://localhost:8080/graphql',
-    cache: new InMemoryCache(),
+  const location = window.location;
+  const params = new URLSearchParams(location.search);
+  const token = params.get('token') as string;
+  const email = params.get('email') as string;
+  const { loading, error, data } = useVerifyTokenQuery({
```

Comment:
You should use lazy version of this hook i.e. `useVerifyTokenLazyQuery`. 
Verify token api should not call very time. It should be call when user coming through invitation email link. We can identify by query param email and token. It will not call when user coming in system through normal login.
Here, either you can call `useVerifyTokenLazyQuery` in useEffect or inside init function by adding email and token check.